config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -41,29 +43,17 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
-        imagePullPolicy: Always
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191106-09649c7-master
         command:
         - runner.sh
         args:
         - ./hack/jenkins/test-dockerized.sh
-        env:
-        - name: KUBE_FORCE_VERIFY_CHECKS
-          value: "n"
-        - name: KUBE_VERIFY_GIT_BRANCH
-          value: master
-        - name: REPO_DIR
-          value: /workspace/k8s.io/kubernetes
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
         resources:
           requests:
             cpu: 4
-    annotations:
```

Comment:
Why remove?

Reply:
https://github.com/kubernetes/test-infra/pull/15144#discussion_r343318791

these annotations were for the canary job to have it's own testgrid entry as a canary, it will be in the same one as the non-master-branch job above